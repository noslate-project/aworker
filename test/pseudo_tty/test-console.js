'use strict';
function foo() {
  console.trace('I', 'am', 'here');
}

function bar() {
  foo();
}

bar();
