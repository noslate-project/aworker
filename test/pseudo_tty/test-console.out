Trace: I am here
    at foo *
    at bar *
    at *test-console.js:10:1
